### PR TITLE
Create a starting point for workflow completeness measurement

### PR DESCRIPTION
**Pull request by mprorock**
based on conversations with several customers, we need more clarity on workflow definition, what credentials are required for particular workflows, and how to identify completeness of a particular workflow

***
<a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/traceability-interop/pull/318.html" title="Last updated on Jul 26, 2022, 2:59 PM UTC (1e1a898)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/traceability-interop/318/409ff5c...1e1a898.html" title="Last updated on Jul 26, 2022, 2:59 PM UTC (1e1a898)">Diff</a>

### CODE REVIEW

**Review by brownoxford — APPROVED** [review]

**Review by nissimsan — APPROVED** [review]

File: docs/index.html [hunk 1]
```diff
@@ -260,20 +260,61 @@ <h3>CCG Standards Stack</h3>
...
+          Workflow <em>instances</em> indicate that separate individual presentations are part of the very same workflow.
         </p>
+          
+        <section class="informative">
+          <h3>Workflows for US Customs and Border Protection (US CBP)</h3>
+          <img src="workflow-def-ex.png" alt="workflow-def-ex" width="75%">
+          <p>
+            In the above example, two separate 7501 Entry
+            Summaries are submitted to US CBP at different points in time, and related to different imports. By specifying the same
+            workflow definition, it is made clear to US CBP that both relate to the same types of import workflows.
```
Comment (TallTed) [inline docs/index.html:277]:
```suggestion
            workflow definition, it is made clear to US CBP that both relate to the same type of import workflows.
```

File: docs/index.html [hunk 2]
```diff
@@ -260,20 +260,61 @@ <h3>CCG Standards Stack</h3>
...
+          </p>
+          <img src="workflow-inst-ex.png" alt="workflow-inst-ex" width="75%">
+          <p>
+            The majority of data elements in a "10+2" Importer Security Filing must be presented to CBP no later than 24 hours before vessel
+            loading, and the remaining data elements no later than 24 hours before arrival. Specifying the same workflow instance indicates
```
Comment (TallTed) [inline docs/index.html:282]:
```suggestion
            loading, and the remaining data elements no later than 24 hours before arrival. Specifying the same workflow <i>instance</i> indicates
```

File: docs/index.html [hunk 3]
```diff
@@ -260,20 +260,61 @@ <h3>CCG Standards Stack</h3>
...
+              Traceability Vocabulary
+            </a>, one per line, and identified by the "type" of credential,
+            for the workflow to be considered "complete".  Completeness
+            in this sense refers to an optimal state where all possible information that
+            can be provided in relation to that workflow definition is present.
+          </p>
+          <p>
+            Following the credential "type" a space, followed by a numeric value that ranges from
+            0 to 1 MAY be added.  If this value is not provided a value of 1 MUST by inferred by
+            any system using this property.  This numeric value SHALL be referred to as "weight",
```
Comment (TallTed) [inline docs/index.html:303]: `SHALL` is technically an RFC 2119 synonym of `MUST`, but infrequently enough used that it is often misinterpreted (or mis-meant) as `SHOULD`, so I would change this `SHALL` to `MUST`.
```suggestion
            any system using this property.  This numeric value MUST be referred to as "weight",
```

**Review by mkhraisha — APPROVED** [review]

File: docs/index.html (hunk 3 above)
Comment (mprorock) [reply docs/index.html:303]: Good catch, thanks

**Pull request merged**